docs/quantum_chess/Quantum_Chess_Client.ipynb

```diff
@@ -0,0 +1,257 @@
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/quantumlib/ReCirq/blob/master/docs/qaoa/example_problems.ipynb\"><img src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" />Run in Google Colab</a>\n",
+        "  </td>\n",
+        "  <td>\n",
+        "    <a target=\"_blank\" href=\"https://github.com/quantumlib/ReCirq/blob/master/docs/qaoa/example_problems.ipynb\"><img src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" />View source on GitHub</a>\n",
+        "  </td>\n",
+        "  <td>\n",
+        "    <a href=\"https://storage.googleapis.com/tensorflow_docs/ReCirq/docs/qaoa/example_problems.ipynb\"><img src=\"https://www.tensorflow.org/images/download_logo_32px.png\" />Download notebook</a>\n",
+        "  </td>\n",
+        "</table>"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "NsP1vVXYuo_-"
+      },
+      "source": [
+        "This is a basic client meant to test the server implemented at the end of the  [Quantum Chess REST API](./quantum_chess_rest_api) documentation. You must run that previous Colab for this one to work."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "zn40ny5Puj1A"
+      },
+      "source": [
+        "!pip install git+https://github.com/quantumlib/ReCirq/ -q\n",
+        "!pip install requests -q"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "glA7YB4VxPDL"
+      },
+      "source": [
+        "The server for the Quantum Chess Rest API endpoints should provide you with an ngrok url when you run it. **Paste the url provided by your server in the form below**. If your server is running, the following code should produce the message: \"Running Flask on Google Colab!\""
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "id": "4JOEEu-ewQ-X"
+      },
+      "source": [
+        "url = 'http://bd626d83c9ec.ngrok.io/' #@param {type:\"string\"}\n",
+        "!curl -s $url"
+      ],
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "pgvtoLl_YltL"
+      },
+      "source": [
+        "You should be able to see the server output indicting a connection was made."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "8GewRWMqGe0G"
+      },
+      "source": [
+        "#Quantum Board Initialization"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "0Lm1rW1TXUzI"
+      },
+      "source": [
+        "We can make a simple request to initialize a board with the starting occupancy state of all pieces. Using the bitboard format, the initial positions of pieces are given by the hex 0xFFFF00000000FFFF. This initializes all squares in ranks 1, 2, 7, and 8 to be occupied."
```

Comment:
```suggestion
        "Make a simple request to initialize a board with the starting occupancy state of all pieces. Using the bitboard format, the initial positions of pieces are given by the hex 0xFFFF00000000FFFF. This initializes all squares in ranks 1, 2, 7, and 8 to be occupied."
```

Google dev docs style prefers "you" instead of "we" though I try to just be direct (and use an implicit "you")